.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+Reason for Change
+===================
+* Describe the big picture of your changes here.
+* This should describe why this pull request should be merged.
+* If it fixes a bug or resolves a feature request, link to that issue.
+
+Changes
+=======
+* Describe your changes as a concise list of items.
+
+Minor
+=====
+* List any minor changes or style fixes here.
+
+Risks
+=====
+* Describe any risks you see to existing systems this fix or feature might have
+if those systems where to incorporate this change.
+
+Checklist
```

Comment:
I would probably not vote to have this section - I do like the "checklist manifesto"-ness of it, but I find it best to have automated things makes these types of checks so I don't have to do the manual work of checking boxes or expecting others to do so (yay overcommit / Circle / code climate!)